hal/armv7m/stm32/l4/config.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Phoenix-RTOS
+ *
+ * plo - operating system loader
+ *
+ * Platform configuration file
+ *
+ * Copyright 2020, 2021 Phoenix Systems
+ * Author: Hubert Buczynski, Aleksander Kaminski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+
+#ifndef _CONFIG_H_
+#define _CONFIG_H_
+
+#ifndef __ASSEMBLY__
+
+#include "stm32l4.h"
+#include "peripherals.h"
+#include "../types.h"
+
+#include <phoenix/arch/syspage-stm32.h>
+#include <phoenix/syspage.h>
+
+#include "../../cpu.h"
+#include "../../mpu.h"
+
+#endif
+
+#define PATH_KERNEL "phoenix-armv7m4-stm32l4x6.elf"
+#define CPU_INFO    "Cortex-M4 STM32L4x6"
+
+#define SIZE_PAGE    0x200
+#define SIZE_STACK   (8 * SIZE_PAGE)
+#define SIZE_SYSPAGE (8 * SIZE_PAGE)
```

Comment:
I see that You took an example from `imxrt106x`. I think that `SIZE_SYSPAGE` can be reduced in Your case. On `stm32` syspage will not be so big.

Reply:
All NOMMU platforms have SIZE_PAGE of 512 bytes, in kernel too. Smaller page actually causes memory to be lost - more memory is needed by page allocator.